packages/design-system-react/src/components/button-base/ButtonBase.test.tsx

```diff
@@ -56,7 +56,7 @@ describe('ButtonBase', () => {
     );
     expect(screen.getByTestId('loading-spinner')).toBeInTheDocument();
     expect(screen.getByText('Please wait...')).toBeInTheDocument();
-    expect(screen.queryByText('Submit')).not.toBeInTheDocument();
+    expect(screen.getByText('Submit')).toHaveClass('invisible');
```

Comment:
To ensure the button retains the same size as it does with its children, and with the loading indicator now absolutely positioned, we simply hide the button's children instead of replacing them using the `invisible` classname.